[Unit-7]Basic chapter of video and video processing (#165)
johko/computer-vision-course: chapters/en/Unit 7 - Video and Video Processing/introduction-to-video.mdx

# Introduction

> ## What is a video?

Reviewer: ```suggestion
## What is a Video?

```

> A binary, two-dimensional (2D) representation of visual data is called an image. In essence, a video is a multimedia format that sequentially displays these frames, or images.

Reviewer: ```suggestion
An image is a  binary, two-dimensional (2D) representation of visual data. A video is a multimedia format that sequentially displays these frames or images.

```
Active voice reads easier

Technically speaking, the frames are separate pictures. As a result, storing and playing these frames sequentially at a conventional speed results in the creation of a video, thus giving the illusion of motion (just like a flipbook).

> It is a popular and widely used medium for communicating information, entertainment, and conversation. Videos, photos, are obtained via image-acquisition equipment such as video cameras, smartphones, and so on.

Reviewer: ```suggestion
It is a popular and widely used medium for communicating information, entertainment, and conversation. Videos and photos are obtained via image-acquisition equipment such as video cameras, smartphones, and so on.

```

> ### Aspects of a video

Reviewer: ```suggestion
### Aspects of a Video

```
maybe attributes?

> 1. **Resolution:**

Reviewer: ```suggestion
- **Resolution**

```

> 1. **Resolution:** <br />

Reviewer 2: The line break tag isn't really necessary.

The resolution of a video refers to the number of pixels in each frame or we can also refer to it as the size of each frame in the video. It doesn't need to be a standard size, but there are common sizes for video. Common video resolutions include HD (1280x720 pixels), Full HD (1920x1080 pixels), Ultra HD or 4K (3840x2160 pixels), and so on.
When a video is said to have a resolution of 1920x1080 pixels, it essentially means the video has a width of 1920 pixels and a height of 1080 pixels.
Higher resolution videos have more detail but also require more storage space and processing power.

> 2. **Frame Rate:**

Reviewer: ```suggestion
- **Frame Rate**

```

A video is composed of multiple separate frames, or images. In order to give the impression of motion, these frames are displayed quickly one after the other.
 The number of frames displayed per second is called the "frame rate." Common frame rates include 24, 30, and 60 frames per second (fps) or hertz (general unit for frequency). Higher frame rates result in smoother motion.

> 3. **Bitrate:**

Reviewer: ```suggestion
- **Bitrate**

```

> The quantity of data needed to describe audio and video is referred to as bitrate. Better quality is achieved at higher bitrates, but streaming requires more storage and bandwidth.

Reviewer: ```suggestion
The quantity of data needed to describe audio and video is called bitrate. Better quality is achieved at higher bitrates, but streaming requires more storage and bandwidth.

```

Bitrates for videos are commonly expressed in megabytes per second (mbps) or kilobytes per second (kbps).

> 4. **Codecs:**

Reviewer: ```suggestion
- **Codecs**

```
